src/components/delete-car/Modal.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Modal.css';
+
+const Modal = ({ isOpen, onCancel, onConfirm }) => (
```

Comment:
- [ ] OPTIONAL: When I click on the confirmation modal to cancel a reservation:

![image](https://user-images.githubusercontent.com/57421823/231304061-c41e6492-b575-4b4c-bf81-5dee1506ea4f.png)

I need to click on "confirm" twice, or else the modal won't disappear.

This is just a nitpick, however, if you just make your users click on "confirm"  once, it will be less confusing and the UX will be improved.

Tip:  On the first click, the action is completed in the backend.  It's just the frontend demanding another click.  I can tell this, because on the second click the reservation is no longer found in the db:

![image](https://user-images.githubusercontent.com/57421823/231304359-4cb95aec-f493-4d3a-9ae9-c4d31a9423c3.png)

_This was tested both on Chrome and Firefox._